packages/devtools_app/lib/src/debugger/flutter/console.dart

```diff
@@ -30,7 +33,102 @@ class Console extends StatefulWidget {
 }
 
 class _ConsoleState extends State<Console> {
-  final scrollController = ScrollController();
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder<List<String>>(
+      valueListenable: widget.controller.stdio,
+      builder: (context, lines, _) {
+        // Extract this to a reusable ConsoleWithControls StatelessWidget?
+        return Stack(
+          children: [
+            ConsoleOutput(lines: lines),
+            Container(
+              alignment: Alignment.bottomRight,
```

Comment:
We may want an upper right alignment here, with a bit of padding from the top and right sides. We have some standard sizes in our theme.dart file - here we might use either `defaultSpacing` or `denseSpacing` depending on how they look.

Reply:
I tried controlling that padding, but I think the ButtonBar is doing its own thing. I'll give this a shot, it's easy enough to test!